### PR TITLE
Combine network mapper

### PR DESCRIPTION
**Pull request by Moran-k**
## Motivation:

Integrate network mapper charts from [PR](https://github.com/komodorio/helm-charts/pull/192) 

## Changes:

* Add the network-mapper container into k8s-watcher deployment
* Add the network-mapper-sniffer into k8s-watcher daemonset 
* Move metrics stuff into a subfolder

## Clickup tasks
CU-860qvnx0u
CU-860qtrpvn
CU-860qtrpqn

### CODE REVIEW

File: charts/k8s-watcher/templates/network-mapper/serviceaccount.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{{- if .Values.serviceAccount.create -}}
```
Comment (MickaelAlliel) [inline charts/k8s-watcher/templates/network-mapper/serviceaccount.yaml:1]: This file is confusing, it's not creating a service account but creating the Bindings to the existing k8s watcher service account

File: charts/k8s-watcher/templates/network-mapper/cluster-role.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - apiGroups:
+      - 'apps'
+    resources:
+      - 'deployments'
+      - 'daemonsets'
+      - 'replicasets'
+      - 'statefulsets'
+    verbs:
+      - 'get'
+{{ if or (and (eq .Values.network_mapper.global.allowGetAllResources nil) .Values.network_mapper.allowGetAllResources) .Values.network_mapper.global.allowGetAllResources}}
```
Comment (MickaelAlliel) [inline charts/k8s-watcher/templates/network-mapper/cluster-role.yaml:25]: Can be simplified: (do check the syntax, i freehanded it)
```suggestion
{{ if or (((.Values.network_mapper).global).allowGetAllResources) (((.Values.network_mapper).global).allowGetAllResources}}
```

File: charts/k8s-watcher/templates/network-mapper/serviceaccount.yaml (hunk 1 above)
Comment (Moran-k) [reply charts/k8s-watcher/templates/network-mapper/serviceaccount.yaml:1]: Will rename to roles-binding.yaml

File: charts/k8s-watcher/templates/network-mapper/cluster-role.yaml (hunk 2 above)
Comment (Moran-k) [reply charts/k8s-watcher/templates/network-mapper/cluster-role.yaml]: On second thought - you are right (+1)

**Review by MickaelAlliel — APPROVED** [review]

**Pull request merged**